src/store/main/actions.ts

```diff
@@ -1,11 +1,30 @@
 import { ActionTree } from 'vuex';
+import { LocalStorage } from 'quasar';
 import { StoreInterface } from '../index';
 import { MainStateInterface } from './state';
+import { SelectedNetwork } from '../../../types/custom-types';
+
+const localSavedNetworkVar = 'kaspa-network'; // name of key for saving network in local storage
 
 const actions: ActionTree<MainStateInterface, StoreInterface> = {
   // eslint-disable-next-line
   async getWalletInfo({ commit }, wallet: any) {
-    await wallet.addressDiscovery(); // eslint-disable-line
+    const network = LocalStorage.getItem(localSavedNetworkVar);
```

Comment:
Hey just starting a review off with some questions. what is `network`? network name, api endpoint, or object/serialized string containing both? this stuff looks a little tricky, we could consider wrapping some tests around it.

Reply:
That is of type `SelectedNetwork` which is defined as follows in `custom-types.ts`:

```javascript
export type Network = 'kaspa' | 'kaspadev' | 'kaspareg' | 'kaspatest' | 'kaspasim';

export interface SelectedNetwork {
  name: Network;
  description: string;
  apiBaseUrl: string;
}
```

where `description` is set to `Manual Network` to identify when the user chose manual vs. automatic